lib/board.rb

```diff
@@ -0,0 +1,15 @@
+class Board
+  attr_accessor :spaces
+
+  def initialize
+    @spaces = ['', '', '', '', '', '', '', '', '']
+  end
+
+  def spaces
```

Comment:
Oh, maybe this is the reader - we could make this an attr_reader method.

Reply:
totally! you are so smart!